HTMX Animations

CarloLepelaars opened the pull request:
Basic animations of text and buttons from the [HTMX animation examples](https://htmx.org/examples/animations). Solves #56 

This app covers 3 elements of the animation examples. I think the example on ["Using the View Transition API"](https://htmx.org/examples/animations/#view-transitions) can be its own app since that one is a bit more involved.

CarloLepelaars committed:
HTMX Animations app

Isaac-Flath committed:
Minor refactoring, naming, and commenting.

Isaac-Flath commented:
@CarloLepelaars I did some refactoring and commenting, can you take a look and see what you think?

Also, can you take a look and maybe add some instructional comments to the css part?  Is all the css important or is there ways to simplify it?  (I agree centering makes the demos look 100x better so that's fine - just checking if there's other stuff that's more minor or not).

Either way, I think some instructional comments to explain what they are is good.  I'm trying to be a bit better about having that in new examples, since I did none of that before and people frequently comment to me about it being lacking in the gallery.

CarloLepelaars committed:
Document CSS and code. New thumbnails

CarloLepelaars commented:
@Isaac-Flath, I added additional comments to code and CSS, cleaned up code and made new thumbnails.

In a separate PR we can review which early apps need more instructional comments, especially ones with a lot of undocumented CSS. Happy to help out with that.

As for the thumbnails, could you document in README.md which resolution you ideally want from contributors for thumbnails? Currently I'm guessing a bit.

Isaac-Flath commented:
Awesome thanks so much!
